eng/pipelines/build.yml

```diff
@@ -34,10 +34,17 @@ stages:
       - job: Build
         timeoutInMinutes: 120
         pool:
-          vmImage: windows-2022
+          name: azsdk-pool-mms-ubuntu-2004-general
+          vmImage: "MMSUbuntu20.04"
```

Comment:
Why are we running the build on ubuntu? are there any issues on windows?

Reply:
Yes, Windows images don't support long paths yet: https://github.com/Azure/azure-sdk-tools/issues/2497